### DISCUSSION TITLE
Feed Sharing

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#775] [Q&A] [answered]
Should it be set to open or restricted?

**Comment by kiyote33** [comment, marked as answer]
Hi @Grateful-Viv, whether to allow anyone to use the json feed from your website (Open) or restrict it's use to only those with a unique key (Restricted) is a personal choice that the admin of each website can make. While some suggest that the feed is just one way of publishing information that is public anyway, so maybe it should always be open; others suggest that the use of an open feed on their sites might lead to fewer people visiting and thus they choose restricted (sending their url with the access key to GSO so they can retrieve meetings for the Meeting Guide app).  In the end, each site can choose for themselves.
